test/datalog_tests/simple3.dat

```diff
@@ -11,7 +11,7 @@ modify Rhosts ddlog_std::Some{"n1"} <- Thosts{.up = ddlog_std::Some{true}},
 commit dump_changes;
 
 start;
-modify Rhosts ddlog_std::Some{"n1"} <- Thosts{.up = ddlog_std::None},
+modify Rhosts ddlog_std::Some{"n1"} <- _{.up = ddlog_std::None},
```

Comment:
how come there was a name before?

Reply:
This is not a generated file.  I change struct name to `_` to test that the parser can handle missing struct names.